user_verification/infrastructure/models.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, Integer, ForeignKey, UniqueConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.mysql import JSON, TIMESTAMP, VARCHAR
+
+Base = declarative_base()
+
+
+class UserVerificationModel(Base):
+    __table_name__ = "user_verification"
+    transaction_id: Column("transaction_id", VARCHAR(128), primary_key=True)
```

Comment:
i guess it is not pytohn syntax to assignment

Reply:
changed